### DISCUSSION TITLE
Saving Level.hierarchy without indents, etc.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3755] [Ideas]
Self-explanatory, would help with file size, especially now that map file size is a worry due to foliage and chart/map images. People wanting to edit the hierarchy file can still get the normal formatting using the built-in tool in VS Code and other editors.

**Comment by SDGNelson** [comment]
Changing it to save as a binary file rather than text is actually on my to-do list. The original reason for the text format was merging but it's not practical.
